jkube-kit/config/image/src/main/java/org/eclipse/jkube/kit/config/image/ImageName.java

```diff
@@ -369,5 +369,9 @@ private boolean isRegistryValidPathComponent() {
     // https://github.com/docker/docker/blob/04da4041757370fb6f85510c8977c5a18ddae380/vendor/github.com/docker/distribution/reference/regexp.go#L37
     private static final Pattern TAG_REGEXP = Pattern.compile("^[\\w][\\w.-]{0,127}$");
 
-    private static final Pattern DIGEST_REGEXP = Pattern.compile("^sha256:[a-z0-9]{32,}$");
+    private static final Pattern DIGEST_256_REGEXP = Pattern.compile("[A-Za-z][A-Za-z0-9]*(?:[-_+.][A-Za-z][A-Za-z0-9]*)*:[0-9a-fA-F]{32,}");
+
+    private static final Pattern DIGEST_512_REGEXP = Pattern.compile("[A-Za-z][A-Za-z0-9]*(?:[-_+.][A-Za-z][A-Za-z0-9]*)*:[0-9a-fA-F]{64,}");
```

Comment:
Could you please check if this works? It's taken from [image-spec for SHA-512](https://github.com/opencontainers/image-spec/blob/main/descriptor.md#sha-512):

> When the algorithm identifier is sha512, the encoded portion MUST match /[a-f0-9]{128}/. Note that [A-F] MUST NOT be used here.

```suggestion
    private static final Pattern DIGEST_512_REGEXP = Pattern.compile("sha512:[0-9a-f]{128}");
```

Reply:
Sure, yes this one is working fine.

Reply:
Yes, It works.